rewrite-observability-platform-introduction page (#2145)
giantswarm/docs: src/content/vintage/platform-overview/observability/_index.md

  - What app do you recommend for monitoring?
  - What app do you recommend for logging?
  - What app do you recommend for tracing?
last_review_date: 2024-02-28
---

> Observability is based on four main data sources: __logs__, __metrics__, __traces__ and __profiles__. To cover these needs Giant Swarm provides its customers with a fully managed observability platform provided with 24//7 monitoring and alerting The tools included in the observabily platform are listed below.

Reviewer: ```suggestion
Observability is based on four main data sources: __logs__, __metrics__, __traces__ and __profiles__. To cover these needs, Giant Swarm provides its customers with a fully managed observability platform supported by 24//7 monitoring and alerting The tools included in the observabily platform are listed below.
```

> Observability is based on four main data sources: __logs__, __metrics__, __traces__ and __profiles__. To cover these needs, Giant Swarm provides its customers with a fully managed observability platform supported by 24//7 monitoring and alerting The tools included in the observabily platform are listed below.

Reviewer 2: ```suggestion
Observability is based on four main data sources: __logs__, __metrics__, __traces__ and __profiles__. To cover these needs, Giant Swarm provides its customers with a fully managed observability platform supported by 24//7 monitoring and alerting. The tools included in the observabily platform are listed below.
```

We chose these applications based on our general principles of preferring open source over proprietary solutions, as well as simplicity, performance, and speed over complexity.

If you want to learn more about observability, we recommend that you read [this](https://opentelemetry.io/docs/concepts/observability-primer/)

To summarize, for observability, we recommend and provide these tools:

## Loki for logging

> To store, index and make logs (containers, kubernetes events, audit logs) searchable. You can also install our [Loki App](https://github.com/giantswarm/loki-app/blob/master/README.md) on your clusters.

Reviewer: ```suggestion
To store, index, and make logs (containers, kubernetes events, audit logs) searchable. You can also install our [Loki App](https://github.com/giantswarm/loki-app/blob/master/README.md) on your clusters.
```

Reviewer: IIUC, the implied distinction here is that we run Loki on MCs for logs we collect, but customers must also install Loki themselves if they want to use it for their own log collection. Is that right?
If so, that could be more explicit here as I found it slightly confusing

Author: Well the implied direction is that we will bé able to store their logs if they want to but they  can also install it by themselves if they want to, but maybe that part should be explained better. I'm not sure how though 😂

Reviewer: What is the benefit if they run Loki themselves?

## Prometheus for monitoring

To get all your metrics, process, store and alert on them. Prometheus has become the de-facto standard monitoring tool in the kubernetes ecosystem.

## Grafana for data visualization

To analyze, visualize, and correlate any kind of observability data.

## Tempo for distributed tracing

We are currently offering Grafana Tempo a part of our observability platform on demand and without any availability garantees due to the low priority it has with our customers.

## Pyroscope for continuous profiling

Profiling is quite a new kid on the block when it comes to observability.

> Same as with Grafana Tempo, we are currently offering Grafana Tempo a part of our observability platform on demand and without any availability garantees due to the low priority it has with our customers.

Reviewer 2: ```suggestion
Same as with Grafana Tempo, we are currently offering Grafana Pyroscope a part of our observability platform on demand and without any availability garantees due to the low priority it has with our customers.
```
